workflow/rules/maftools.smk

```diff
@@ -0,0 +1,36 @@
+rule group_bcf_to_vcf:
+    input:
+        "results/final-calls/{group}.{event}.{calling_type}.fdr-controlled.bcf",
+    output:
+        temp("results/maftools/{group}.{event}.{calling_type}.fdr-controlled.vcf"),
+    log:
+        "logs/maftools/{group}.{event}.{calling_type}.fdr-controlled.log",
+    params:
+        extra="",
+    wrapper:
+        "v3.8.0/bio/bcftools/view"
+
+
+rule group_vcf_to_maf:
+    input:
+        vcf="results/maftools/{group}.{event}.{calling_type}.fdr-controlled.vcf",
+        ref=genome,
+    output:
+        maf="results/maftools/{group}.{event}.{calling_type}.fdr-controlled.maf",
+    log:
+        "logs/maftools/{group}.{event}.{calling_type}.fdr-controlled.log",
+    conda:
+        "../envs/vcf2maf.yaml"
+    params:
+        genome_build=config["ref"]["build"],
+        species=config["ref"]["species"],
+    shell:
+        "vcf2maf.pl --inhibit-vep "
+        " --retain-info PROB_ARTIFACT,PROB_ABSENT,PROB_SOMATIC,PROB_GERMLINE_HOM,PROB_GERMLINE_HET "
```

Comment:
It is not guaranteed that users have these events in their scenario. I guess it makes sense to infer that from the input bcf.

Reply:
Yes, this is now done via params functions. This only took a 3 hour refactoring and then a 5 hour cleanup, but it is definitely better this way. We should seriously think about modularizing this workflow -- and making variant calling and fusion calling separate modules. Having both in one makes it harder to maintain.